app/collections/files.js

```diff
@@ -300,5 +305,24 @@ module.exports = Backbone.Collection.extend({
 
   url: function() {
     return this.repo.url() + '/git/trees/' + this.sha + '?recursive=1';
+  },
+
+  getVariables: function() {
+    var login = cookie.get('login');
+
+    var user = login;
+    if (this.config && this.config.users) {
```

Comment:
This should probably check `if (this.config && Array.isArray(this.config.users))` to make sure it doesn't throw if someone mis-entered this field?

Reply:
Done